### PR TITLE
Detect use of asterisk (*) in PackageReference items of .NET projects

### PR DESCRIPTION
**Pull request by realmarv**

### CODE REVIEW

**Comment by knocte** [comment]
Let's update the ReadMe as part of this PR.

**Comment by realmarv** [comment]
> Let's update the ReadMe as part of this PR.

Done

**Comment by knocte** [comment]
Let's rebase this PR to get rid of the red CI.

**Comment by realmarv** [comment]
> Let's rebase this PR to get rid of the red CI.

Done

File: scripts/detectUnpinnedVersions.fsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    |> Seq.filter(NotInDir "bin")
+    |> Seq.filter(NotInDir "obj")
+    |> Seq.filter(NotInDir "DummyFiles")
+    |> Seq.filter(fun fileInfo ->
+        FileConventions.DetectAsteriskInPackageReferenceItems fileInfo
+    )
+
+if Seq.length invalidFiles > 0 then
+    let message =
+        "The following files shouldn't use asterisk (*) in PackageReference items of .NET projects:"
```
[the remark this answers is not in the document]
Comment (realmarv) [reply scripts/detectUnpinnedVersions.fsx]: Done

**Pull request merged**